manifests/init.pp

```diff
@@ -53,6 +53,7 @@
   $group               = 'root',
   $listen_address      = undef,
   $repo                = undef,
+  $repo_package_name   = undef,
```

Comment:
you need here a reasonable default or derive value from `params.pp`

Reply:
In fact, I think a more elegant way to versionize Mesos is via your $mesos::version property. 

I overlooked it. Once again, thank you very much.